src/index.js

```diff
@@ -20,11 +18,7 @@ $('#submit').click(event => {
   if ($('#username').val() === 'manager' && $('#password').val() === 'overlook2019') {
     window.location = './manager.html'
   }
-
-  if ($('#username').val().includes('customer')) {
-    userID = parseInt($('#username').val().split('r')[1]);
-    localStorage.setItem('userID', userID);
-  }
+  let userID = parseInt($('#username').val().slice(-2));
```

Comment:
Interesting change here.  Slicing from -2 instead of splitting AND no longer saving to local storage.

Can I ask why you decided switch it?

I just ran into something in my code where I had to implement local storage.  I didn't think I would have to since I'm only using one index.js file and my class files but I couldn't seem to get around it.  

I couldn't get anything after my `window.location` line to fire which is why I had to implement local storage.
How did you manage to not need it?

Reply:
I did a huge rework that I ended up control z a lot of stuff and it looks like I went to far in this file, saved me another bug catch! Thank you!